system/infra-monitoring/templates/_prometheus-infra-collector.yaml.tpl

```diff
@@ -976,14 +976,13 @@
 - job_name: 'esxi-host'
   scrape_interval: {{$values.scrapeInterval}}
   scrape_timeout: {{$values.scrapeTimeout}}
-  file_sd_configs:
-      - files :
-        - /etc/prometheus/configmaps/atlas-netbox-sd/netbox.json
+  kubernetes_sd_configs:
+  - role: service
+    namespaces:
+      names:
+      - infra-monitoring
   metrics_path: /
   relabel_configs:
-    - source_labels: [job]
-      regex: esxi
-      action: keep
     - target_label: __address__
```

Comment:
please use respective variable from the available meta services. `address` is not containing the service name.
https://prometheus.io/docs/prometheus/latest/configuration/configuration/#service